zipline/gens/tradesimulation.py

```diff
@@ -190,10 +201,34 @@ def on_exit():
             if algo.data_frequency == 'minute':
                 def execute_order_cancellation_policy():
                     algo.blotter.execute_cancel_policy(DAY_END)
+
+                def process_minute_capital_changes(dt):
```

Comment:
Does defining this function require the processor or `ZiplineAPI` context to have been entered?

It is surprising that it is defined after `once_a_day` is defined.